app/services/degrees/create_from_hesa.rb

```diff
@@ -31,6 +33,8 @@ def call
 
     def create_degrees!
       hesa_degrees.map do |hesa_degree|
+        next if non_importable?(hesa_degree)
```

Comment:
Although it works it feels a bit hard to read. Maybe we could rename the method for clarity?
```suggestion
        next unless importable?(hesa_degree)
```

And then something, maybe, like this?
```ruby
    def importable?(hesa_degree)
      !hesa_degree[:degree_type].include?(UNKNOWN_DEGREE_TYPE) && hesa_degree.compact.empty?
    end
```

Reply:
Looks good, I'll have a think how to reverse the check. I don't think the above check quite works as we can accept 999 degree types so long as one other piece of information is not nil. But will have a think!